.github/CODEOWNERS

```diff
@@ -30,813 +30,810 @@
+# ServiceLabel: %Application Insights
+# ServiceOwners:                                     @azmonapplicationinsights
 
-# AzureSdkOwners: @anuchandy @conniey @lmolkova
-# ServiceLabel: %Azure.Core.AMQP
-# PRLabel: %Azure.Core.AMQP
-/sdk/core/azure-core-amqp/                           @anuchandy @conniey @lmolkova
+# ServiceLabel: %App Services
+# ServiceOwners:                                     @antcp @AzureAppServiceCLI
 
-# AzureSdkOwners: @lmolkova
-# ServiceLabel: %OpenTelemetry
-# PRLabel: %OpenTelemetry
-/sdk/core/azure-core-tracing-opentelemetry/          @lmolkova @trask
+# ServiceLabel: %ARM
+# ServiceOwners:                                     @armleads-azure
 
-# ServiceLabel: %Cosmos
-# PRLabel: %Cosmos
-/sdk/cosmos/                                         @kushagraThapar @FabianMeiswinkel @kirankumarkolli @xinlian12 @milismsft @aayush3011 @simorenoh @jeet1995 @Pilchie
+# ServiceLabel: %ARM - Templates
+# ServiceOwners:                                     @armleads-azure
 
-# PRLabel: %Load Testing
-/sdk/loadtesting/                                    @Harshan01 @prativen
+# ServiceLabel: %ARM - Tags
+# ServiceOwners:                                     @armleads-azure
 
-# ServiceLabel: %Device Update for IoT Hub %Service Attention
-/sdk/deviceupdate/                                   @dpokluda
+# ServiceLabel: %ARM - Core
+# ServiceOwners:                                     @armleads-azure
 
-# PRLabel: %Digital Twins
-/sdk/digitaltwins/                                   @johngallardo @efriesner @abhinav-ghai @Aashish93-stack @sjiherzig @Satya-Kolluri
+# ServiceLabel: %ARM - Managed Applications
+# ServiceOwners:                                     @armleads-azure
 
-# PRLabel: %Models Repository
-/sdk/modelsrepository/                               @drwill-ms @timtay-microsoft @abhipsaMisra @digimaun @andyk-ms @brycewang-microsoft @tmahmood-microsoft @ngastelum-ms
+# ServiceLabel: %ARM - Service Catalog
+# ServiceOwners:                                     @armleads-azure
 
-# AzureSdkOwners: @billwert
-# ServiceOwners:                                     @jfggdl
-# ServiceLabel: %Event Grid
+# ServiceLabel: %ARM - RBAC
+# ServiceOwners:                                     @armleads-azure
 
-# PRLabel: %Event Grid
-/sdk/eventgrid/                                     @billwert @mssfang @srnagar @lmolkova
+# ServiceLabel: %ARO
+# ServiceOwners:                                     @mjudeikis @jim-minter @julienstroheker @amanohar
 
-# AzureSdkOwners: @conniey @anuchandy @lmolkova
-# ServiceOwners:                                    @serkantkaraca @sjkwak 
-# ServiceLabel: %Event Hubs
+# AzureSdkOwners: @srnagar
+# ServiceLabel: %graalvm
+# PRLabel: %graalvm
+/sdk/aot/                                            @srnagar @jonathangiles
 
-# PRLabel: %Event Hubs
-/sdk/eventhubs/                                      @conniey @anuchandy @lmolkova
+# PRLabel: %Attestation
```

Comment:
This PRLabel applies only to the first path beneath it.  You'll need to put one on top of each path individually.